### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/knative-networking/commit/71c77c99df37b2db19e0ffd4341353002fd85afc#commitcomment-43154247

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/knative-networking/commit/754747a46d80b2ac5ebc1c44ec84bf03dc2e7470

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: pkg/client/injection/reconciler/networking/v1alpha1/certificate/reconciler.go [hunk 1]
```diff
@@ -183,7 +183,7 @@ func (r *reconcilerImpl) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 
 	// Initialize the reconciler state. This will convert the namespace/name
-	// string into a distinct namespace and name, determin if this instance of
+	// string into a distinct namespace and name, determine if this instance of
```
Comment (jsoref) [inline pkg/client/injection/reconciler/networking/v1alpha1/certificate/reconciler.go:186]: It's possible this comes from somewhere. Let me know and I'll try to send a patch to it at some point.

Comment (jsoref) [reply pkg/client/injection/reconciler/networking/v1alpha1/certificate/reconciler.go]: https://github.com/knative/pkg/pull/1797

File: pkg/apis/networking/v1alpha1/ingress_types.go [hunk 2]
```diff
@@ -185,7 +185,7 @@ type IngressRule struct {
 type HTTPIngressRuleValue struct {
 	// A collection of paths that map requests to backends.
 	//
-	// If they are multiple matching paths, the first match takes precendent.
+	// If they are multiple matching paths, the first match takes precedent.
```
Comment (vagababov) [inline pkg/apis/networking/v1alpha1/ingress_types.go:188]: this must be `precedence`

File: pkg/apis/networking/v1alpha1/serverlessservice_lifecycle.go [hunk 3]
```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var serverlessServiceCondSet = apis.NewLivingConditionSet(
-	ServerlessServiceConditionEndspointsPopulated,
+	ServerlessServiceConditionEndpointsPopulated,
```
Comment (vagababov) [inline pkg/apis/networking/v1alpha1/serverlessservice_lifecycle.go:29]: Don't rename conditions, please.

**Review by vagababov — COMMENTED** [review]
modulo comments otherwise looks good.

File: pkg/apis/networking/v1alpha1/serverlessservice_lifecycle.go (hunk 3 above)
Comment (jsoref) [reply pkg/apis/networking/v1alpha1/serverlessservice_lifecycle.go]: Dropped

**Review by vagababov — APPROVED** [review]
/lgtm

**Pull request merged**